libs/payments/paypal/src/lib/client.ts

```diff
@@ -122,8 +121,49 @@ export class PayPalClient {
       return resultObj;
     } else {
       // TypeScript doesn't narrow ACK, necessitating a cast
-      throw new PayPalClientError(result.text, resultObj as NVPErrorResponse);
+      throw new MultiError(
+        this.getListOfPayPalClientErrors(
+          result.text,
+          resultObj as NVPErrorResponse
+        )
+      );
+    }
+  }
+
+  public getListOfPayPalClientErrors(raw: string, data: NVPErrorResponse) {
+    const errors: PayPalClientError[] = [];
+    if (!data.L || !data.L.length) {
+      const message =
+        'PayPal NVP returned a non-success ACK. See "this.raw" or "this.data" for more details.';
```

Comment:
This isn't a user facing error message, right?

Reply:
No